### DISCUSSION TITLE
Cannot find module slugify (no valid entry point) from dependecies wich uses v1.6.5,  v1.6.4 works though

### DISCUSSION THREAD

**Discussion by leonvanderstel** [gatsbyjs/gatsby#34507] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

A clean gatsby installation (^4.5.2) breaks while running gatsby develop

Gatsby uses V1.6.5 of slugify. V1.6.5 doesn't have a slugify.js file. Which breaks gatsby because it cannot find the slugify.js entry point V1.6.5 slugify is from 12 days ago on time of writing

![screen](https://user-images.githubusercontent.com/74352577/149622382-1df4e7fd-3b78-41bc-86ea-8acf5c0fa4ae.png)

Build error in my case gatsby js context
![screen2](https://user-images.githubusercontent.com/74352577/149622048-b935e094-7229-47d9-bc78-a0260a74c772.png)

When installing v1.6.4 within node_modules/gatsby everything works and there is a slugify.js wich it can refer to. 

![screen3](https://user-images.githubusercontent.com/74352577/149622250-68e010bf-b004-4d08-b9f9-2a0f1eeb21be.png)

Maybe downgraden to 1.6.4 till 1.6.5 works?

### Reproduction Link

Clean install, check steps to reproduce

### Steps to Reproduce

From clean install

- npm gatsby init
- gatsby develop

### Expected Result

run gatsby develop and everything builds as expected to run in browser

### Actual Result

build step breaks while running gatsby develop

### Environment

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  Binaries:
    Node: 14.15.3 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.9 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 2.7.15
  Browsers:
    Chrome: 97.0.4692.71
    Edge: Spartan (44.19041.1266.0), Chromium (97.0.1072.62)
  npmPackages:
    gatsby: ^4.5.2 => 4.5.2 
    gatsby-cli: ^4.5.1 => 4.5.1 
  npmGlobalPackages:
    gatsby-cli: 3.14.0
```

**Comment by LekoArts** [comment]
Please follow the advice given here: https://github.com/simov/slugify/issues/149#issuecomment-1013764193
